distribution/src/test/resources/invalid-hazelcast.yaml

```diff
@@ -0,0 +1 @@
+configuration: invalid
```

Comment:
Is this used in any test?

Reply:
A leftover from removed `HazelcastMemberTest`, removed https://github.com/hazelcast/hazelcast/pull/20262/commits/caad3027fa2d8c7da796dd7fc520a5ff5a26e958